firmware/src/util/actions.cpp

```diff
@@ -173,46 +176,40 @@ bool low_level_channel_three(int16_t state) {
 // Same as servo 2 but digital output
 bool low_level_channel_four(int16_t state) {
   if (state == 0 || state == 1) {
-    servo_set_onoff(&SERVO2, (bool)state);
+    // servo_set_onoff(&SERVO2, (bool)state);
     return true;
   }
   return false;
 }
 
 // Servo Outputs
 
-bool servo_channel_one(int16_t angle) {
-  if (angle >= 0 && angle <= 180) {
-    servo_set_position(&SERVO1, angle);
-    return true;
+bool servo_channel_one(int16_t position) {
+  if (global_servo1 == nullptr) {
+    return false;
   }
-  return false;
-}
-
-bool servo_channel_two(int16_t angle) {
-  if (angle >= 0 && angle <= 180) {
-    servo_set_position(&SERVO2, angle);
+  if (position >= 0 && position <= 1000) {
+    global_servo1->SetPosition(position);
     return true;
   }
   return false;
 }
 
-bool servo_channel_three(int16_t angle) {
-  if (angle >= 0 && angle <= 180) {
-    // servo_set_position(&SERVO1, angle);
-    return true;
+bool servo_channel_two(int16_t position) {
+  if (global_servo1 == nullptr) {
+    return false;
   }
-  return false;
-}
-
-bool servo_channel_four(int16_t angle) {
-  if (angle >= 0 && angle <= 180) {
-    // servo_set_position(&SERVO1, angle);
+  if (position >= 0 && position <= 1000) {
+    global_servo2->SetPosition(position);
     return true;
   }
   return false;
 }
 
+bool servo_channel_three(int16_t position) { return false; }
```

Comment:
Why do we have four servo channels? We can only connect two, no?

Reply:
Will remove everything that is unused soon